apiserver/facades/client/application/privileges.go

```diff
@@ -0,0 +1,42 @@
+package application
+
+import (
+	"github.com/juju/errors"
+	"gopkg.in/juju/environschema.v1"
+)
+
+const privilegeConfigOptionName = "elevated_privileges"
```

Comment:
I believe most config strings are "-" separated. Just like "juju bootstrap --config http-proxy" this should be "elevated-privileges".

Reply:
The name is going to be changed to "trust" for both the option and flag. This is just a spike to show how implementing on top of config might look.